client/java/build.gradle

```diff
@@ -135,10 +135,18 @@ publishing {
 
     repositories {
         maven {
-            url = 'https://oss.sonatype.org/service/local/staging/deploy/maven2'
-            credentials {
-                username = System.getenv('SONATYPE_USER')
-                password = System.getenv('SONATYPE_PASSWORD')
+            if (isReleaseVersion) {
+                url = 'https://oss.sonatype.org/service/local/staging/deploy/maven2'
+                credentials {
+                    username = System.getenv('SONATYPE_USER')
+                    password = System.getenv('SONATYPE_PASSWORD')
+                }
+            } else {
+                url = 'https://datakin.jfrog.io/artifactory/maven-public-libs-snapshot'
+                credentials {
+                    username = System.getenv('ARTIFACTORY_USERNAME')
+                    password = System.getenv('ARTIFACTORY_PASSWORD')
+                }
```

Comment:
I think we can manage this switching in the CI, rather than in the gradle build

Reply:
Yes, we can manage it via context, for https://oss.sonatype.org/service/local/staging/deploy/maven2' we can use exist 'release' context, for https://datakin.jfrog.io/artifactory/maven-public-libs-snapshot we can create a new context like 'snapshot'. Then we also need common variable name like PUBLISH_USERNAME /  PUBLISH_PASSWORD which we will manage via different context. Are you ok with that and can you help to manage that?

Reply:
Yeah, I think having a common env variable makes sense. You can set the common variables from circle ci- see [this example](https://circleci.com/docs/2.0/env-vars/?utm_medium=SEM&utm_source=gnb&utm_campaign=SEM-gnb-DSA-Eng-ni&utm_content=&utm_term=dynamicSearch-&gclid=CjwKCAjwxo6IBhBKEiwAXSYBs20tejw_8UMAqcLsqXwxinCO8DkENPv08q_G3aW1VmQZlyK-Svq_OhoCnbkQAvD_BwE#example-configuration-of-environment-variables)

Reply:
I'm having error when opening circleCI/OpenLineage/Organization settings.
_Something Unexpected Happened
This is certainly annoying, our apologies. Maybe a refresh would help. If that doesn't work, you may not have permissions to view this page or you may have the incorrect URL._
@julienledem  maybe I need some additional permissions for that? Cause it works for Marquez